habitat-lab/habitat/config/habitat/task/ovmm/gaze.yaml

```diff
@@ -9,6 +9,7 @@ defaults:
     - robot_force
     - force_terminate
     - robot_colls
+    - robot_collisions_terminate
```

Comment:
@yvsriram does this look fine? There are two collision measures here so just wanted to make sure.

Reply:
Yeah, by default these are not effective.